src/linearmixedmodel.jl

```diff
@@ -417,7 +417,7 @@ end
 
 Return the names of the grouping factors for the random-effects terms.
 """
-fnames(m::MixedModel) = (fname.(m.reterms)...,)
+fnames(m::MixedModel) = ((tr.trm.sym for tr in m.reterms)...,)
```

Comment:
Looks like this change was problematic.  I will revert it.

Reply:
See #361